doc/ChangeLog

```diff
@@ -1,4 +1,231 @@
+
+Changes to CTSM:
+================
+
+All new surface datasets. Transient urban and lake by default turned on for transient cases.
+Ocean is run as baresoil rather than wetland (for clm6_0). The urban streams file was also
+updated. The new surface datasets have new updated input datasets (see below).
+
+New surface datasets:
+=====================
+
+The new surface datasets are incompatible with previous versions (for example the ctsm5.1 series)
+ctsm5.2.0 and following versions also can NOT use the previous ctsm5.1 datasets.
+
+See the section below about the new datasets used in their creation. Improvements in how landunits
+on coastal areas was also done.
+
+The following fields were added and removed to the list of fields on the datasets.
+
+Fields added:
+   ORGC, BULK, CFRAG, PHAQ (soil data)
+   mapunits (map units for the soil dataset)
+   LANDFRAC_MKSURFDATA
+   PCT_OCEAN (previously PCT_WETLAND was used)
+
+Fields removed:
+   AREA, PFTDATA_MASK
+
+New mksurfdata_esmf Tool:
+=========================
+
+Implement a parallel version of mksurfdata (mksurfdata_esmf) that uses ESMF regridding directly
+in mksurfdata so that offline mapping files don't have to be created as a separate step. This
+allows mksurfdata to create surface datasets at much higher resolutions
+
+The build for the tool is based on the CESM/CIME build system and uses cmake. This allows the build
+to be kept up with changes in CESM. Currently it's only setup and working on Derecho. But, this design
+will enable it to be built and run on any CESM supported machine (or a machine that a user ports to).
+
+Any input grid from ccs_config can be used, or the user can supply their own mesh file to define
+the output grid. The user no longer has to add to the list of valid resolutions as in mksurfdata_map.
+
+Creation of supported single point datasets. These datasets are created through the use of subset_data.
+
+Test datasets for dynUrban, dynLake, and dynPFT is done with a simple NCO script.
+
+All datasets can be easily made by running "make all" in the tools/mksurfdata_esmf directory.
+
+For instructions see:
+    tools/mksurfdata_esmf/README.md
+
+New input datasets to mksurfdata_esmf:
+======================================
+
+New soil dataset: ISRIC/WISE dataset Batjes (2016) https://doi.org/10.1016/j.geoderma.2016.01.034
+New PFT, soil-color, LAI dataset: Created by Lawrence P.J. 2022
+New Glacier datasets: Glacier outlines from RGI version 6 (Arendt et al., 2017).
+   vector data for GrIS and AIS retrieved from BedMachine version 4 and version 2 (Morlighem et al., 2017, 2020), respectively.
+   30-arcsec topography/land mask retrieved GMTED2010 (Danielson and Gesch, 2011).
+New urban datasets:  Gao and O'Neill (2021) and Gao and Pesaresi (2022), Oleson and Feddema (2020)
+New lake datasets: HydroLake: Messager et. al. (2016)
+
+Contributors
+------------
+@mvertens @ekluzek @jedwards4b @billsacks @wwieder @lawrencepj1 @negin513 @dlawrenncar @olyson
```

Comment:
Probably @slevis-lmwg goes third or so.

Reply:
Resolved. Also added the urban dataset contributors...